package.json

```diff
@@ -62,6 +62,7 @@
     "is-mergeable-object": "1.1.0",
     "isparta": "4.0.0",
     "jest": "22.0.4",
+    "jest-cli": "22.0.4",
```

Comment:
:+1: it looks that is this explicit dependency on jest-cli that is fixing the issue

I think that we can remove the "jest" dependency above it, so that we don't need to keep them in sync when we upgrade jest (I tried locally and it worked fine).

Reply:
Hmm, that's actually true but given that `jest` only has `jest-cli` as a dependency we don't have to pin anything right now. Once they pin `jest-cli` we can get rid of the explicit dependency.

Reply:
sounds good